Разметка внутренней страницы

OliaErmak opened the pull request:
Разметка каталога и добавила алреса ссылок в соц сети на главной страничке

---
:mortar_board: [Разметка внутренней страницы](https://up.htmlacademy.ru/htmlcss/33/user/315091/tasks/4)

:boom: https://htmlacademy-htmlcss.github.io/315091-device-33/3/

OliaErmak committed:
Добавляет адреса ссылок

Добавила адреса для ссылок соц сетей в футере

OliaErmak committed:
Добавляет разметку каталога

ad0v0 commented on catalog.html:
```diff
@@ -0,0 +1,190 @@
+            </section>
+          </section>
+
+          <div class="download-more">
```
заменить на show-more, тк это не та загрузка, которая загрузит какой-то файл на комп пользователю

ad0v0 commented:
**Проверка разметки главной и каталога:**

1) Проверка задания согласно критериям:
- ✅ [PROJ-01. Кодгайд](https://up.htmlacademy.ru/htmlcss/32/criteries#PROJ-01) - ок 
- ✅ [PROJ-02. Именование](https://up.htmlacademy.ru/htmlcss/32/criteries#PROJ-02) - ок 
- ✅ [PROJ-03. Название папок и файлов](https://up.htmlacademy.ru/htmlcss/32/criteries#PROJ-03):
- ✅ [HTML-01. Ориентиры](https://up.htmlacademy.ru/htmlcss/32/criteries#HTML-01) - ок 
- ✅ [HTML-02. Заголовки](https://up.htmlacademy.ru/htmlcss/32/criteries#HTML-02) - ок
- ✅ [HTML-03. Кнопки и ссылки](https://up.htmlacademy.ru/htmlcss/32/criteries#HTML-03):
- ✅ [HTML-04. Работоспособные формы](https://up.htmlacademy.ru/htmlcss/32/criteries#HTML-04):
- ✅ [HTML-05. Подписи полей форм](https://up.htmlacademy.ru/htmlcss/32/criteries#HTML-05) - ок 
- ✅ [HTML-07. Валидация HTML](https://up.htmlacademy.ru/htmlcss/32/criteries#HTML-07) - ок 
- ✅ [IMG-01. Контентная и декоративная графика](https://up.htmlacademy.ru/htmlcss/32/criteries#IMG-01) - ок 
- ✅ [CONT-01. Текстовые переносы](https://up.htmlacademy.ru/htmlcss/32/criteries#CONT-01) - ок 
- ✅ [ALLY-01. Описание графики](https://up.htmlacademy.ru/htmlcss/32/criteries#ALLY-01) - ок 
- ☑️ [ALLY-02. Подписи интерактивных элементов](https://up.htmlacademy.ru/htmlcss/32/criteries#ALLY-02):
-- http://joxi.ru/L21BGwXtwnlZQA добавить в разметку сортировку по приоритету и затем описание 
2) Разное:
- ☑️ http://joxi.ru/BA0YGw1S196d5r заменить input на button type submit и убрать подпись


🎌 - есть замечание, надо править
➡️ - есть комментарий, желательно учесть
✅ - выполнено 
☑️ - правка по замечанию внесена
❓ - есть вопрос, желательно ответить

ПС Ошибки , которые дублируются в коде, в замечаниях не дублирую, но перепроверять и править нужно везде и обязательно

ad0v0 commented on catalog.html:
```diff
@@ -0,0 +1,190 @@
+        <h2 class="visually-hidden">Социальные сети</h2>
+        <a class="social" href="https://www.facebook.com/htmlacademy"><span class="visually-hidden">Facebook</span></a>
+        <a class="social" href="https://www.instagram.com/htmlacademy/"><span class="visually-hidden">Instagram</span></a>
+        <a class="social" href="https://twitter.com/htmlacademy_ru"><span class="visually-hidden">Twitter</span></a>
```
сокнопки можно обернуть в список. понимаешь почему?

ad0v0 commented on catalog.html:
```diff
@@ -0,0 +1,190 @@
+            <option value="popular">Популярное</option>
+          </select>
+
+          <section class="cards">
```
Предлагаю использовать сделать следующую структуру в разметке:
```
section.catalog
  ul.catalog__list
    li.catalog__item
      article.product
```
Что нам это даст:
1) разбивка стилизации по, так сказать, функционалу: по каталогу строим сетку, а по продукту карточку с товаром
2) у нас появляется два независимых друг от друга компонента: компонент с каталогом, который может содержать все, что угодно (от карточек с товаром до фоток); компонент с карточкой, которую можно повторно использовать на любой странице на сайте или даже вынести с сайта (кстати, здесь еще салютуем тегу article, который показывает, что контент внутри него вполне самостоятелен и понятен вне контекста конкретного=текущего расположения)

OliaErmak committed:
Исправляет разметку #4

Главная:
1) Добавила форму Доставка под заказ

Главная и каталог:
2) Добавила список в соц сети на главной и каталоге

Каталог:
3) Заменила download-more на show-more
4) Заменила input type="submit" на button type="submit"
5) Добавила ссылки на странички в пагинации
6) Добавила сортировку по приоритету
7) Изменила структуру в section class="catalog"

ad0v0 commented:
есть, принято!
